### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by atian25** [comment]
顺便改下 https://github.com/eggjs/egg/pull/1979 里面遗漏的 en 版

File: History.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
 
+## 2018-01-26, Version 2.3.0, @dead-horse
+
+### Notable changes
+
+* **feature**
+  * emit `request` and `response` event in every request
+
+* **docs**
+  * improve english docs
```
Comment (atian25) [inline History.md]: https://github.com/eggjs/egg/pull/1979

Comment (dead-horse) [reply History.md]: 1979的有问题，应该是用 exports 啊，我给他改回去了..

Comment (atian25) [reply History.md]: 额... 也是，没写全的。

**Pull request merged**